src/validator/CreditCardVendorReading.java

```diff
@@ -0,0 +1,7 @@
+package validator;
+
+import java.util.ArrayList;
+
+public interface CreditCardVendorReading {
```

Comment:
this looks like a repository. I'd rename this to `CreditCardVendorRepository` or `CreditCardVendorReader`